### ISSUE TITLE
[Android] Always getting error "Last found location is older than maximumAge (FusedLocationProvider/lastLocation)"

### ISSUE THREAD

**Issue by jvalbhani** [michalchudziak/react-native-geolocation#211]
## Versions

- react-native-geolocation: 3.0.3
- react-native: 0.69

## Description

Was facing Bug#206 in 3.0.2 
after updating to 3.0.3 always getting below error
location: {“TIMEOUT”:3,“POSITION_UNAVAILABLE”:2,“PERMISSION_DENIED”:1,“message”:“Last found location is older than maximumAge (FusedLocationProvider/lastLocation).“,”code”:2}

**Comment by ifarhanpatel** [comment]
When cached location is older than maximumAge, shouldn't we request location updates instead of returning an error?

**Comment by michalchudziak** [comment]
Thank you for reporting, I'll look onto it when I will have some time :) Feel free to send a PR fixing the behavior as well :)

**Comment by michalchudziak** [comment]
> When the cached location is older than the maximum age, shouldn't we request location updates instead of returning an error?

That's definitely the way to go. I'll ship the fix soon, it should work properly on 3.0.4.

**Comment by rumax** [comment]
Issue still exists:

```
$ git diff package.json
diff --git a/package.json b/package.json
index 1d5fe89..7f01ce7 100644
--- a/package.json
+++ b/package.json
@@ -18,7 +18,7 @@
   },
   "dependencies": {
     "@react-native-async-storage/async-storage": "^1.17.11",
-    "@react-native-community/geolocation": "^3.0.3",
+    "@react-native-community/geolocation": "^3.0.4",
```

gives 

```
{"PERMISSION_DENIED": 1, "POSITION_UNAVAILABLE": 2, "TIMEOUT": 3, "code": 2, "message": "Last found location is older than maximumAge (FusedLocationProvider/observer)."}
```

Used config:

```
{
        interval: 100,
        maximumAge: 100,
        enableHighAccuracy: true,
        distanceFilter: 0,
        useSignificantChanges: false,
}
```

if I comment `maximumAge` location is returned once, but updates do not happen.
